utils/rpms/daos.spec

```diff
@@ -586,6 +586,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Thu Feb 08 2024 Dalton Bohning <dalton.bohning@intel.com> 2.5.100-15
```

Comment:
Does this warrant a new version number?  All you're doing is adding a new file.

Reply:
My understanding is anything that changes the binary/code/library result of the RPM needs a version bump.

@brianjmurrell Could you please confirm whether a `Release` bump is necessary for a new file?

Reply:
@ashleypittman also pointed out that this new test didn't bump the release
https://github.com/daos-stack/daos/pull/13681/files

Reply:
Modifications to daos.spec needing a release number seems like a good guideline but in the case of this PR then only changing the python substitution would be a case where I don't see any benefit.

Reply:
@brianjmurrell I will defer to you on whether I need to bump the release here, per Ashley's comment

Reply:
It's up to you @daltonbohning.  The least friction option would be of course not to bump.

Reply:
Thanks. I took the least friction route :)